202305_dc0/combine_maps.py

```diff
@@ -0,0 +1,97 @@
+import os
+import healpy as hp
+import numpy as np
+from astropy.table import QTable
+from pathlib import Path
+
+extragalactic = ["cib_cib1", "ksz_ksz1", "tsz_tsz1", "radio_rg1"]
+all_combined = {
+    "combined_cmb_unlensed_dipole": ["cmb_c3", "dipole"],
+    "combined_cmb_lensing_signal": ["cmb_c4", "-cmb_c3"],
+    "combined_foregrounds_mediumcomplexity": [
+        "dust_d10",
+        "synchrotron_s4",
+        "freefree_f1",
+        "ame_a1",
+        "co_co3",
+    ]
+    + extragalactic,
+    "combined_foregrounds_lowcomplexity": [
+        "dust_d9",
+        "synchrotron_s5",
```

Comment:
Just double-checking: does `synchrotron_s4` have higher complexity than `synchrotron_s5`? For all other components the complexity seems to increase with the version number.

Reply:
thanks, this is a bug, I launched the script again, will let you know when this is transferred to nersc

Reply:
Ok transferred